packages/ruleset/src/functions/property-case-collision.js

```diff
@@ -0,0 +1,32 @@
+const { validateSubschemas } = require('../utils');
+
+module.exports = function(schema, _opts, { path }) {
+  return validateSubschemas(schema, path, propertyCaseCollision);
+};
+
+const errorMsg = 'Property names should not be identical, even if different naming conventions are used';
```

Comment:
My first reaction to this error message is that 'identical' might not be the right word, precisely speaking.   I think "duplicate" might be a better word in this case since duplicate implies that the two properties are the same in some aspect but not necessarily "identical".
```suggestion
const errorMsg = 'Avoid duplicate property names within a schema, even if different case conventions are used';
```